rgw/v2/tests/s3cmd/reusable.py

```diff
@@ -56,6 +57,32 @@ def upload_file(bucket_name):
     return file_name
 
 
+def download_file(bucket_name, remote_file_name, local_file_name=None):
+    """
+    Downloads file from the bucket
+    Args:
+        bucket_name(str): Name of the bucket
+        remote_file_name(str): Name of the remote file
+        local_file_name(str): Name of the local file to be set
+    Returns: Name of the downloaded file
+    """
+    if local_file_name is None:
```

Comment:
Likewise we can avoid this check if the set the default value of `local_file_name` to __test_s3cmd.txt__ instead of `None`.

Reply:
Same as above. I added it to have functionality of giving customized name as the method is called twice in same testcase.

Reply:
Understood.